### DISCUSSION TITLE
Access unauthorized 401

### DISCUSSION THREAD

**Discussion by happytonny** [langgenius/dify#9237] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I used dify to publish a workflow and accessed it through the API. I referred to the following API documentation:
curl -X POST 'http://localhost/v1/workflows/run' \
--header 'Authorization: Bearer {app-G6c5inhYNOkk1cLXAWIByAlp}' \
--header 'Content-Type: application/json' \
--data-raw '{
    "inputs": {},
    "response_mode": "streaming",
    "user": "abc-123"
}'
When accessing, the following error is reported all the time: {"code": "unauthorized", "message": "Access token is invalid", "status": 401}.

**Comment by siddiqodiq** [comment]
--header 'Authorization: Bearer app-70fiCwaJ2cSrWJMsOwy28GA9' \
--header 'Content-Type: application/json' \ 

delete {}

**Reply by QiQi-OvO** [reply]
I've removed {}, but 401 again :(
![image](https://github.com/user-attachments/assets/47cca2a0-0320-437c-8f44-18da160e2103)

**Comment by fanfan-lucky** [comment]
Do you resolve this issue? I met the same problem....If you got the answer, please tell me and thank you.

**Comment by Nyoko74** [comment]
I had the same problem. Have you solved it?

**Comment by fanfan-lucky** [comment]
[cid:05720552-77de-496e-b3eb-570fd7822363]
________________________________

**Reply by zjxiegj** [reply]
I have the same problem。help！！
{"code":"unauthorized","message":"Access token is invalid","status":401}

**Comment by zjxiegj** [comment]
I have the same problem。help！！
{"code":"unauthorized","message":"Access token is invalid","status":401}

**Comment by zjxiegj** [comment]
I have the same problem。help！！
{"code":"unauthorized","message":"Access token is invalid","status":401}
